### ISSUE TITLE
CMake configure regression when targeting Xcode/iOS

### ISSUE THREAD

**Issue by fwcd** [mixxxdj/mixxx#13378]
### Bug Description

The most recent `main` branch can no longer be configured for Xcode (i.e. with `cmake -G Xcode ...`):

```
CMake Error in res/shaders/CMakeLists.txt:
  The custom command generating

    /Users/<user>/git/m1xxx-ios/mixxx/build/res/shaders/.rcc/qrc_waveform_shaders.cpp

  is attached to multiple targets:

    mixxx-lib_waveform_shaders
    mixxx-lib_resources_1

  but none of these is a common dependency of the other(s).  This is not
  allowed by the Xcode "new build system".


CMake Generate step failed.  Build files cannot be regenerated correctly.
```

### Version

https://github.com/fwcd/mixxx/commit/218517904ab03d776ecd0fb8cb7bbe3dd5b87850 (based on c368119262913411f94766a3e59735678bd72832)

### OS

macOS 14.5

**Comment by fwcd** [comment]
This looks very similar to https://bugreports.qt.io/browse/QTBUG-103723. Still pretty strange, since the iOS build environment, which is based on our 2.5 vcpkg environment, uses Qt 6.5.3, which should supposedly already contain the fix.

**Comment by fwcd** [comment]
This might actually be https://bugreports.qt.io/browse/QTBUG-118500 instead.

**Comment by JoergAtGithub** [comment]
Can this be closed now, after #13379 got merged?

**Comment by fwcd** [comment]
Yes, thanks for spotting this, no idea why the PR linking didn't work
